src/intl/en/page-what-is-ethereum.json

```diff
@@ -61,26 +58,24 @@
-  "page-what-is-ethereum-who-runs-ethereum-desc-2": "These distributed nodes, run by individuals and businesses all over the world, provide resiliency to the Ethereum network infrastructure. It is therefore much less vulnerable to hacks or shutdowns. <b>Since its launch in 2015, Ethereum has never suffered downtime</b>. There are thousands of individual nodes running Ethereum network. This makes Ethereum one of the most decentralized cryptocurrencies out there, second only to bitcoin.",
+  "page-what-is-ethereum-who-runs-ethereum-desc-1": "Ethereum is not controlled by any particular entity. It exists whenever there are connected computers running software following the Ethereum protocol and adding to the Ethereum blockchain. Each of these computers is known as a node. Nodes can be run by anyone, although to participate in securing the network you have to stake ETH (Ethereum’s native token). Anyone with 32 ETH can do this without needing permission.",
+  "page-what-is-ethereum-who-runs-ethereum-desc-2": "Even the Ethereum source code is not produced by a single entity. Anyone can suggest changes to the protocol and discuss upgrades. There are several versions implementations of the Ethereum protocol that are produced by independent organizations in several programming languages, and they are usually built in the open and encourage community contributions.",
   "page-what-is-ethereum-run-a-node": "Run a node",
   "page-what-is-ethereum-smart-contract-title": "What are smart contracts?",
-  "page-what-is-ethereum-smart-contract-desc-1": "<b>Smart contracts are simply computer programs</b> living on the Ethereum blockchain. They only <b>execute when triggered by a transaction from a user (or another contract)</b>. They make Ethereum very flexible in what it can do and distinguish it from other cryptocurrencies. These programs are what we now call decentralized apps, or dapps.",
+  "page-what-is-ethereum-smart-contract-desc-1": "<b>Smart contracts are computer programs</b> living on the Ethereum blockchain. They execute when triggered by a transaction from a user. They make Ethereum very flexible in what it can do. These programs act as building blocks for decentralized apps and organizations.",
   "page-what-is-ethereum-smart-contract-desc-2": "Have you ever used a product that changed its terms of service? Or removed a feature you found useful? Once a smart contract is published to Ethereum, it will be online and operational for as long as Ethereum exists. Not even the author can take it down. Since smart contracts are automated, they do not discriminate against any user and are always ready to use.",
   "page-what-is-ethereum-smart-contract-desc-3": "Popular examples of smart contracts are lending apps, decentralized trading exchanges, insurance, crowdfunding apps - basically anything you can think of.",
```

Comment:
```suggestion
  "page-what-is-ethereum-smart-contract-desc-3": "Popular examples of smart contracts are lending apps, decentralized trading exchanges, insurance, quadratic funding, social networks, NFTs - basically anything you can think of.",
```

I know this wasn't changed in this version, but think it may be worth expanding a bit of this here.